x/auth/tx/service.go

```diff
@@ -37,6 +39,83 @@ func NewTxServer(clientCtx client.Context, simulate baseAppSimulateFn, interface
 
 var _ txtypes.ServiceServer = txServer{}
 
+const (
+	eventFormat = "{eventType}.{eventAttribute}={value}"
+)
+
+// TxsByEvents implements the ServiceServer.TxsByEvents RPC method.
+func (s txServer) TxsByEvents(ctx context.Context, req *txtypes.GetTxsEventRequest) (*txtypes.TxsByEventsResponse, error) {
+
+	if req.Page < 0 {
+		return nil, status.Error(codes.InvalidArgument, "page must greater than 0")
+	}
+	if req.Limit < 0 {
+		return nil, status.Error(codes.InvalidArgument, "limit must greater than 0")
+	}
+	if len(req.Event) == 0 {
+		return nil, status.Error(codes.InvalidArgument, "must declare at least one event to search")
+	}
+	var events []string
+	if strings.Contains(req.Event, "&") {
+		events = strings.Split(req.Event, "&")
```

Comment:
is the `msg1.sender=...&msg2.sender=...` with `&` standard, or just something used in CLI?

If the latter, maybe we could use `repeated string events = 1;` in the proto definition?

Reply:
It's a standard for `Get` APIs (REST at least).

Reply:
Sounds good, let keep it like this then 👍